UnitTests/GitCommandsTests/GitModuleTest.cs

```diff
@@ -504,6 +504,41 @@ public void GetSubmodulesLocalPaths()
             }
         }
 
+        [Test]
+        public void GetSuperprojectCurrentCheckout()
+        {
+            // Create super and sub repo
+            var moduleTestHelperSuper = new CommonTestUtils.GitModuleTestHelper("super repo");
+            var moduleTestHelperSub = new CommonTestUtils.GitModuleTestHelper("sub repo");
```

Comment:
⚠️ `GitModuleTestHelper` is disposable, please wrap these two in `using`.

Reply:
Done